src/components/Input/Input.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+type InputProps = {
+  onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
лучше сразу сюда передавать, и `onChange` должен быть обязательным
```suggestion
  onChange: (value: string) => void;
```

Reply:
а внутри инпута делать `const handleChange = (e) => {onChange(e.target.value)}` `onChange={handleChange}`